cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-user-uuids.cy.js

```diff
@@ -137,5 +137,24 @@ describe('bulk-edit', () => {
       BulkEditActions.openInAppStartBulkEditFrom();
       BulkEditActions.verifyCalendarItem();
     });
+
+    it('C359585 Verify clicking on the "Commit changes" button (firebird)', { tags: [testTypes.smoke, devTeams.firebird] }, () => {
+      BulkEditSearchPane.selectRecordIdentifier('User UUIDs');
+
+      BulkEditSearchPane.uploadFile(userUUIDsFileName);
+      BulkEditSearchPane.waitFileUploading();
+
+      BulkEditActions.openActions();
+      BulkEditActions.verifyCheckedDropdownMenuItem();
+      BulkEditActions.verifyUncheckedDropdownMenuItem();
+
+      BulkEditActions.openInAppStartBulkEditFrom();
+      BulkEditActions.fillPatronGroup('graduate (Graduate Student)');
+      BulkEditActions.confirmChanges();
+      BulkEditActions.verifyAreYouSureForm(1, user.username);
+      BulkEditActions.commitChanges();
+      BulkEditSearchPane.waitFileUploading();
+      BulkEditActions.verifySuccessBanner(1);
```

Comment:
It is better to check if the value is been updated also. There is a note about it in the expected result column on the step 8 in the test case: The Preview of record changed is shown with only updated values

Reply:
done